Artsy_Tests/View_Controller_Tests/SaleOnHoldOverlayViewTests.swift

```diff
@@ -62,7 +62,7 @@ class SaleOnHoldOverlayViewTests: QuickSpec {
                 subject.align(toView: containerView!)
             }
 
-            xit("handles subsequent message updates") {
+            it("handles subsequent message updates") {
```

Comment:
```suggestion
            xit("handles subsequent message updates") {
```
not worth fighting for.